doc/extra-large-conference/README.md

```diff
@@ -0,0 +1,88 @@
+WARNING: This is still a Work In Progress
+
+The final implementation may diverge. Currently, only the participants after a
+configured threshold will be just viewers (visitors) and there is no promotion
+mechanism to become a main participant.
+
+TODO:
+* Merge messaging between visitor nodes and main conference
+* Polls
+* Raise hand to be promoted to enter the main conference
+* Make sure it works with tenants.
+
+
+# Low-latency conference streaming to very large audiences
+
+To have a low-latency conference with a very large audience, the media and
+signaling load must be spread  beyond what can be handled by a typical Jitsi
+installation. A call with 10k participants requires around 50 bridges on decent
+vms (8+ cores). The main participants of a conference with a very large
+audience will share a main prosody, like with normal conferences, and
+additional prosody vms are needed to support signaling to the audience.
+
+In the example configuration we use a 16 core machine. Eight of the cores are
+used for the main prosody and other services (nginx, jicofo, etc) and the other
+eight cores are used to run prosody services for visitors, i.e., "visitor
+prosodies".
+
+We consider 2000 participants per visitor node a safe value. So eight visitor
+prosodies will be enough for one 10k participants meeting.
+
+<img src="imgs/visitors-prosody.svg" alt="diagram of a central prosody connected to several visitor prosodies" width="500"/>
```

Comment:
Do we have the source for the svg so it's easy to edit or is it just inkscape'd?

Reply:
The svg is in the PR.